src/Pages/ContentListTable/components/StatusIcon.tsx

```diff
@@ -22,13 +37,71 @@ const useStyles = createUseStyles({
     margin: '-3px 0',
   },
 });
+
 interface Props {
-  status?: string;
+  rowData: ContentItem;
+  retryHandler?: (string) => Promise<void>;
+}
+
+interface DescriptionProps {
   error?: string;
+  count?: number;
+  time?: string;
+}
+
+interface FooterProps {
+  retryHandler?: (string) => Promise<void>;
+  uuid: string;
 }
 
-const StatusIcon = ({ status, error }: Props) => {
+const PopoverDescription = ({ error, count, time }: DescriptionProps) => {
+  const timeText = time === '' || time === undefined ? 'Never' : dayjs(time).fromNow();
+  return (
+    <div>
+      {error}
+      <Flex>
+        <Stack className='pf-u-mt-sm'>
+          <StackItem className='pf-u-font-weight-bold'>Last introspection</StackItem>
+          <StackItem> {timeText} </StackItem>
+        </Stack>
+        <Stack className='pf-u-mt-sm'>
+          <StackItem className='pf-u-font-weight-bold'>Failed attempts</StackItem>
+          <StackItem> {count} </StackItem>
+        </Stack>
+      </Flex>
+    </div>
+  );
+};
+
+const PopoverFooter = ({ retryHandler, uuid }: FooterProps) => {
+  const { rbac } = useAppContext();
```

Comment:
This is what is making your test fail (I think). You'll want to mock the useAppContext to return this value.